site/src/sphinx/client-custom-http-headers.rst

```diff
@@ -1,9 +1,13 @@
-Sending custom HTTP headers with a Thrift call
-==============================================
-To send a custom HTTP header such as authentication token with a Thrift call, you can:
+.. _client-custom-http-headers:
 
-- use the ``Clients.withHttpHeaders()`` method or
-- use the ``ClientOption.HTTP_HEADERS`` option.
+Sending custom HTTP headers
+===========================
+To send a custom HTTP header such as authentication token with a Thrift or GRPC call, you can:
+
+- use the ``Clients.withHttpHeaders()`` method,
```

Comment:
I saw other page doesn't use comma and `or` to write the list, so it would be better to be same as below titles?
```
- Using ``Clients.withHttpHeaders()``
- Using ``ClientOption.HTTP_HEADERS``
- Using ``ClientBuilder.decorator()``
- Using a derived client
```